### PR TITLE
Add CI build for Android Image Classification example

### PR DESCRIPTION
**Pull request by edgchen1**
Add CI build and simple test that loads a model.
Rename directory from image_classifications to image_classification.

### CODE REVIEW

File: ci_build/python/run_android_emulator.py [hunk 1]
```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python3
```
Comment (edgchen1) [inline ci_build/python/run_android_emulator.py:1]: FYI, this (and the scripts it depends on) was mostly copied from here:
https://github.com/microsoft/onnxruntime/blob/0888c6cc5926ac5debf85b105264933784591b82/tools/python/run_android_emulator.py

File: mobile/examples/image_classification/android/app/src/androidTest/java/ai/onnxruntime/example/imageclassifier/ExampleInstrumentedTest.kt [hunk 2]
```diff
@@ -19,4 +21,17 @@ class ExampleInstrumentedTest {
...
+    fun loadModel() {
+        // Context of the app under test.
+        val appContext = InstrumentationRegistry.getInstrumentation().targetContext
+        val resources = appContext.getResources()
+        val modelBytes = resources.openRawResource(R.raw.mobilenet_v2_float).readBytes()
+        val env = OrtEnvironment.getEnvironment()
+        val session = env?.createSession(modelBytes)
+        assertNotNull(session)
+        session?.close()
+        env?.close()
```
Comment (gwang-msft) [inline mobile/examples/image_classification/android/app/src/androidTest/java/ai/onnxruntime/example/imageclassifier/ExampleInstrumentedTest.kt:35]:
```suggestion
        val env = OrtEnvironment.getEnvironment()
        env.use {
          assertNotNull(env)
          val session = env.createSession(modelBytes)
          session.use {
            assertNotNull(session)
          }
        }
```

**Review by gwang-msft — APPROVED** [review]

File: mobile/examples/image_classification/android/README.md [hunk 3]
```diff
@@ -17,12 +17,12 @@ We use pre-trained [TorchVision MOBILENET V2](https://pytorch.org/hub/pytorch_vi
...
 
 ### Step 0. [Optional] Prepare the ORT models
-Open [Mobilenet v2 Quantization with ONNX Runtime Notebook](https://github.com/microsoft/onnxruntime/blob/master/onnxruntime/python/tools/quantization/notebooks/imagenet_v2/mobilenet.ipynb), this notebook will demostrate how to,
+Open [Mobilenet v2 Quantization with ONNX Runtime Notebook](https://github.com/microsoft/onnxruntime-inference-examples/blob/main/quantization/notebooks/imagenet_v2/mobilenet.ipynb), this notebook will demonstrate how to,
 1. Export the pre-trained MobileNet V2 FP32 model from PyTorch to a FP32 ONNX model
 2. Quantize the FP32 ONNX model to an uint8 ONNX model
 3. Convert both FP32 and uint8 ONNX models to ORT models
 
-Note: this step is optional, you can download the FP32 and uint8 ORT models [here](https://1drv.ms/u/s!Auaxv_56eyubgQlfGIBWh-j1wYyl?e=GYKoL7).
+Note: this step is optional, you can download the FP32 and uint8 ORT models [here](https://onnxruntimeexamplesdata.blob.core.windows.net/examplesdata/mobilenet_v2_ort_models.zip).
```
Comment (gwang-msft) [inline mobile/examples/image_classification/android/README.md:26]: Add run `download_model_files.sh` as another way to add model and label to step 1?

File: mobile/examples/image_classification/android/app/src/androidTest/java/ai/onnxruntime/example/imageclassifier/ExampleInstrumentedTest.kt (hunk 2 above)
Comment (edgchen1) [reply mobile/examples/image_classification/android/app/src/androidTest/java/ai/onnxruntime/example/imageclassifier/ExampleInstrumentedTest.kt:35]: nice, thanks!

**Pull request merged**